exasol_integration_test_docker_environment/lib/api/spawn_test_environment.py

```diff
@@ -79,5 +95,10 @@ def spawn_test_environment(
                                               is_setup_database_activated=not deactivate_database_setup,
                                               create_certificates=create_certificates
                                               )
-    success, task = run_task(task_creator, workers, task_dependencies_dot_file)
-    return success
+    try:
+        result = run_task(task_creator, workers, task_dependencies_dot_file)
+        environment_info = result[DEFAULT_RETURN_OBJECT_NAME]
+        return environment_info, functools.partial(_cleanup, environment_info)
```

Comment:
Hm, maybe later we should add on top of this api a context manager

Reply:
created #243 for that purpose.